pkg/storage/storage_provisioning.go

```diff
@@ -53,6 +53,8 @@ type ProvisionedBucket struct {
 	Name string
 	// Endpoint
 	Endpoint string
+	// Geography of the resource
+	Geography string
```

Comment:
Rename to Region to be consistent with the account

Reply:
Done Thanks